src-tauri/src/commands.rs

```diff
@@ -73,12 +78,13 @@ pub async fn disconnect(
         disconnect_interface(&connection, &state).await?;
         debug!("Connection saved");
         handle.emit_all(
-            "connection-changed",
+            CONNECTION_CHANGED,
             Payload {
                 message: "Created new connection".into(),
             },
         )?;
         stop_log_watcher_task(&handle, &interface_name)?;
+        maybe_update_instance_config(location_id, &handle).await?;
```

Comment:
```suggestion
        if connection_type == ConnectionType::Location {
            maybe_update_instance_config(location_id, &handle).await?;
        }
```

seems like passing tunnel id as location_id will always result in an error